tilecloud_chain/tests/test_error.py

```diff
@@ -180,7 +178,7 @@ def test_wrong_sequence(self, l):
             'tilecloud_chain', 'ERROR', """The config file 'tilegeneration/wrong_sequence.yaml' is invalid.
  - /: Cannot find required key 'caches'.
  - /: Cannot find required key 'layers'.
- - /grids/test/resolutions: Value 'test' is not a list. Value path: '/grids/test/resolutions'
+ - /grids/test/resolutions: Value 'b'test'' is not a list. Value path: '/grids/test/resolutions'
```

Comment:
:-/

Reply:
This message is not built by us... So I cannot fix it.